components/base/kinematicbase/execution.go

```diff
@@ -29,7 +29,7 @@ const (
 
 	// Used to determine minimum linear deviation allowed before correction attempt. Determined by multiplying max linear speed by
 	// inputUpdateStepSeconds, and will correct if deviation is larger than this percent of that amount.
-	minDeviationToCorrectPct = 50.
+	minDeviationToCorrectPct = 70.
```

Comment:
I don't think we want to change this as part of this PR; let's just do the logs + the bugfix.